docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -118,6 +118,17 @@ To learn more, refer to the following video demo.
 
 {{< video-embed src="/media/docs/grafana/2023-09-11-New-Browse-Dashboards-Enablement-Video.mp4" >}}
 
+## Multiple spansets per trace
+
+<!-- Joey Tawadrous -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+The [TraceQL query editor](https://grafana.com/docs/tempo/latest/traceql/#traceql-query-editor) has been improved to facilitate the grouping of multiple spansets per trace in TraceQL queries. For example, when the following `by(resource.service.name)` is added to your TraceQL query it will group the spans in each trace by `resource.service.name`.
```

Comment:
```suggestion
The [TraceQL query editor](https://grafana.com/docs/tempo/<TEMPO VERSION>/traceql/#traceql-query-editor) has been improved to facilitate the grouping of multiple spansets per trace in TraceQL queries. For example, when the following `by(resource.service.name)` is added to your TraceQL query, it will group the spans in each trace by `resource.service.name`.
```

Reply:
This can be https://grafana.com/docs/tempo/v2.2.x/traceql/#traceql-query-editor

Reply:
That sounds great, feel free to commit! :) Thank you